libwallet/api_impl_types.go

```diff
@@ -20,6 +20,11 @@ import (
 	"github.com/blockcypher/libgrin/libwallet/slateversions"
 )
 
+type AccountPathMapping struct {
+	Label string
+	Path  string
+}
+
```

Comment:
If we follow Grin structure this should be in `libwallet/types.go` https://github.com/mimblewimble/grin-wallet/blob/master/libwallet/src/types.rs#L907 also can you add the comment about what it is?